dataset/dictionary/characters.json5

```diff
@@ -1676,10 +1694,10 @@
     ],
   },
   {
-    en: "Captain of Swordfish Ⅱ",
-    ja: "メカジキ二番隊隊長",
-    zhCN: "剑鱼二番队队长",
-    pronunciationJa: "メカジキにばんたいたいちょう",
+    en: "Swordfish Ⅱ",
+    ja: "メカジキ二番隊",
+    zhCN: "剑鱼二番队",
+    pronunciationJa: "メカジキにばんたい",
```

Comment:
@Bill-Haku @SleepyAsh0191 I'm removing "Captain of" from "Captain of Swordfish Ⅱ". Can you review if the Chinese translation is properly updated?

Reply:
Yes that's correct

Reply:
@Bill-Haku Thanks for the quick confirmation!